assets/scss/03-organisms/_campaign-video.scss

```diff
@@ -0,0 +1,33 @@
+.ma__campaign-video {
+  padding: 100px 0;
+  text-align: center;
+
+  &--wrapper {
+    @include ma-container();
+  }
+
+  &--bay-blue {
+    background-color: rgba($c-bay-blue, 0.5);
```

Comment:
Are these color specified by the design team?
This blue doesn't seem to have enough color contrast against white text.